openassessment/workflow/models.py

```diff
@@ -180,21 +180,6 @@ def score(self):
         """
         return sub_api.get_latest_score_for_submission(self.submission_uuid)
 
-    @property
-    def override_score(self):
-        """
-        Latest override score.
-        
-        Note an override score has no submission associated with it.
```

Comment:
Did you do an analysis of how these overrides are stored in the db and if there is any discrepancy between the old and new storage. In effect, what will happen to existing grade overrides in the new code; do they need a migration? (https://github.com/Stanford-Online/edx-submissions/pull/1)